xAPI.md

```diff
@@ -438,6 +439,16 @@ __Registration__: An instance of a learner experiencing a particular Activity.
 __Representational State Transfer (REST)__: An architecture for designing networked web Services.
 It relies on HTTP methods and uses current web best practices.
 
+<a name="def-reporting-system" />
+
+__Reporting System__: A system or tool used to report on and analyse xAPI data and display this to the user. 
```

Comment:
Not sure I understanding the question, but will have a go. 

The requirements relating to this kind of system are about what you should display to the user vs. what the system should know about e.g. voiding statements.